rust/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasi-nn"
-version = "0.6.0"
+version = "25.0.2"
```

Comment:
Can we just move this version to `0.7.0`?

Reply:
Updated

Reply:
Thanks! I think @jlb6740 is going to take a look at this PR further; thanks for the contribution!